.vib/opencart/cypress/cypress/integration/opencart_spec.js

```diff
@@ -1,28 +1,11 @@
 /// <reference types="cypress" />
 import { random } from '../support/utils';
 
-it('allows a user to register', () => {
-  cy.visit('/index.php?route=account/login');
-  cy.contains('Continue').click();
-  cy.fixture('users').then((user) => {
```

Comment:
I think you missed deleting the related fixtures' files no longer used. You should be able to remove the `users/downloads.json` and the `images` folder.

Reply:
You are right, I totally forgot! Removed 🚀